zeebe/exporters/rdbms-exporter/src/main/java/io/camunda/exporter/rdbms/handlers/JobExportHandler.java

```diff
@@ -52,12 +58,47 @@ public void export(final Record<JobRecordValue> record) {
   }
 
   private JobDbModel map(final Record<JobRecordValue> record) {
-    return new Builder()
-        .jobKey(record.getKey())
-        .processInstanceKey(record.getValue().getProcessInstanceKey())
-        .retries(record.getValue().getRetries())
-        .state(record.getIntent().name())
-        .partitionId(record.getPartitionId())
-        .build();
+    final JobRecordValue value = record.getValue();
+
+    final Builder builder =
+        new Builder()
+            .jobKey(record.getKey())
+            .partitionId(record.getPartitionId())
+            .processInstanceKey(value.getProcessInstanceKey())
+            .elementInstanceKey(value.getElementInstanceKey())
+            .processDefinitionId(value.getBpmnProcessId())
+            .processDefinitionKey(value.getProcessDefinitionKey())
+            .tenantId(value.getTenantId())
+            .type(value.getType())
+            .worker(value.getWorker())
+            .state(JobState.valueOf(record.getIntent().name()))
+            .retries(value.getRetries())
+            .errorMessage(value.getErrorMessage())
+            .errorCode(value.getErrorCode())
+            .endTime(DateUtil.toOffsetDateTime(Instant.ofEpochMilli(record.getTimestamp())))
```

Comment:
💭 I believe we should only set this when the intent is `JobIntent.COMPLETED`. Would you agree, @korthout? If so, we should probably fix this in the Camunda Exporter as well (where I assume you were inspired from, @tsedekey?).

Reply:
Yes I think so too. Do we know what the `endTime` is used for? 

If it describes the time that the job was completed (the end time), then we should fill it on `COMPLETED` (and we could consider `CANCELED`). For all other intents, the timestamps is also filled, but the job has not yet reached the end of its lifecycle. 

If it describes the time of any latest update (the last updated time), then we can just always fill it with timestamp.

Reply:
💭 I would suggest to just make the change. I don't expect anyone to assume `endTime` to mean latest updated moment. So, if anything it's a bug fix :D

The alternative is to figure out where we call this API and how the value is used. I don't know if we have any easy tricks for this. But it would be great if we had something to simplify such checks.

Reply:
I tend to agree here 👍 

@tsedekey, let's fix this here and in the Camunda Exporter in the same way (looks like we're running into maintenance issues with multiple exporters that do the same thing basically 🎉 ).
We only set the `endTime` if the intent is COMPLETED or CANCELED.

Let's also gather feedback in [#team-orchestration-cluster](https://camunda.slack.com/archives/C08F5RD5X8B) on the usage of this attribute `endTime`. There can be a) UI usages and b) exported record handling in other components, I believe. Let's see if anyone disagrees with doing this.

Reply:
>and in the Camunda Exporter in the same way (looks like we're running into maintenance issues with multiple exporters that do the same thing basically

💭 I expected that the Camunda Exporter just has different targets ES/OS/RDMS. Seems that's assumption was wrong 😢

Reply:
Yep, I thought so as well in the beginning 🥲